api/api.pb.txt

```diff
@@ -9175,6 +9192,10 @@ file {
       name: "ConfChangeUpdateNode"
       number: 2
     }
+    value {
+      name: "ConfChangeAddLearnerNode"
```

Comment:
This was added in https://github.com/etcd-io/etcd/pull/8751, related to https://github.com/etcd-io/etcd/issues/8568

wondering if we need local changes to take the new type into account (e.g. https://github.com/docker/swarmkit/pull/1711)

(should be ok as follow up)

Reply:
cc @dperny